### PR TITLE
Document Dockerfile

### PR DESCRIPTION
**Pull request by 1084565**
In this repo's docker branch we're having a Dockerfile. It is a copy (no
changes!) of the original in devops-docker-images/cm-client

Tasks:

* [ ] protect the docker branch (just like master)
* [ ] set up docker build
* [ ] delete Dockerfile from devops/docker-images

Question:
* Do we want Dockerfile linting (Travis?)

### CODE REVIEW

**Comment by 1084565** [comment]
Please review. Dockerfile (hidden in the `docker` branch has no docuemntation so far and no build + the branch is not protected.

File: README.md [hunk 1]
```diff
@@ -28,7 +28,20 @@ the actions necessary within those scenarios. See section _Usage_ for more detai
 
 This command line client can be consumed either as a Java application from [maven.org](http://repo1.maven.org/maven2/com/sap/devops/cmclient/dist.cli) or as a Docker image from [hub.docker.com](https://hub.docker.com/r/ppiper/cm-client).
 
-## Installation from maven.org
+
+## Using the Docker Image
+On a linux machine you can run
```
Comment (SarahNoack) [inline README.md]:
```suggestion
On a linux machine you can run:
```

File: README.md [hunk 2]
```diff
@@ -28,7 +28,20 @@ the actions necessary within those scenarios. See section _Usage_ for more detai
...
+## Using the Docker Image
+On a linux machine you can run 
+
+`docker run --rm ppiper/cm-client cmclient --help`
+
+This will print the CM Client's help information. For a comprehensive overview of available commands please read the [usage information](#usage) below.
+
+### How to Build the Docker Image
+
+The Dockerfile is located in a designated branch [`dockerfile`](/SAP/devops-cm-client/tree/docker). After checking out the branch you can run
```
Comment (SarahNoack) [inline README.md]:
```suggestion
The Dockerfile is located in a designated branch [`dockerfile`](/SAP/devops-cm-client/tree/docker). After checking out the branch you can run:
```

File: README.md [hunk 3]
```diff
@@ -28,7 +28,20 @@ the actions necessary within those scenarios. See section _Usage_ for more detai
...
+`docker build -t cm-client .`
+
+## Using the Java Application from maven.org
 
   - Download the command line interface package from [maven.org](http://repo1.maven.org/maven2/com/sap/devops/cmclient/dist.cli)
```
Comment (SarahNoack) [inline README.md]:
```suggestion
  - Download the command line interface package from [maven.org](http://repo1.maven.org/maven2/com/sap/devops/cmclient/dist.cli).
```

File: README.md [hunk 4]
```diff
@@ -28,7 +28,20 @@ the actions necessary within those scenarios. See section _Usage_ for more detai
...
   - Extract the command line interface package into suitable folder
```
Comment (SarahNoack) [inline README.md]:
```suggestion
  - Extract the command line interface package into suitable folder.
```

**Comment by 1084565** [comment]
Please review @radsoulbeard

**Pull request merged**